src/components/inputfield.jsx

```
@@ -1,77 +1,189 @@
+                                            variant="outlined"
+                                            placeholder="you@email.com"
+                                            name="email"
+                                            value={values.email}
+                                            onChange={handleChange("email")}
+                                            sx={{
+                                                borderRadius: "10px",
+                                                backgroundColor: "white",
+                                                width: "100%",
+                                                '& .MuiOutlinedInput-root': {
+                                                    '& fieldset': {
+                                                        border: isFocused ? '1px solid transparent' : '1px solid transparent',
```

Biome

[error] This property value named border is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named border


(lint/suspicious/noDuplicateObjectKeys)